assets/composer/settings.php

```diff
@@ -754,6 +754,12 @@
 if (file_exists($app_root . '/' . $site_path . '/settings.local.php')) {
   include $app_root . '/' . $site_path . '/settings.local.php';
 }
-elseif (file_exists($app_root . '/' . $site_path . '/settings.lando.php')) {
+elseif (getenv('LANDO_SERVICE_NAME') == 'appserver' && file_exists($app_root . '/' . $site_path . '/settings.lando.php')) {
```

Comment:
I have a LANDO = ON, but can't find it in the documentation, which would feel more the equivalent of IS_DDEV_PROJECT. Still chances of anyone changing the prefix 'LANDO_' or the service name 'appserver' and not understanding this is minimal.